src/components/Header/nati_staff.tsx

```diff
@@ -0,0 +1,59 @@
+import { ContactNDiv, FullName, Icon, Name, NArrow, NDiv, NTextArea, Role, SkillsHeather, SkillsList } from "./style";
+
+interface WhichPageProps {
+  onMainPage: boolean;
+}
+
+export const NatiInformation = (props: WhichPageProps) => {
+  const handleInIconClick = () => {
+    window.open("https://www.linkedin.com/in/natalia-mateusiak/", "_blank");
+  };
+  const handleGithubIconClick = () => {
+    window.open("https://github.com/natimati", "_blank");
+    };
+    
+    const NSkills = () => {
+      return (
+        <SkillsList isBeti={false}>
+          <li>HTML</li>
+          <li>CSS, Styled Components</li>
+          <li>JavaScript</li>
+          <li>ReactJS</li>
+          <li>TypeScript</li>
+          <li>WebSockets and Rest API</li>
+          <li>GitHub</li>
+        </SkillsList>
+      );
+    };    
+
+  if (!props.onMainPage) {
```

Comment:
It's good you are using if + return instead of nesting if condition in single return 👍🏻